packages/quantic/cypress/e2e/default-2/standalone-search-box/standalone-search-box-selectors.ts

```diff
@@ -12,18 +13,28 @@ export interface StandaloneSearchBoxSelector extends ComponentSelector {
 
 export const StandaloneSearchBoxSelectors: StandaloneSearchBoxSelector = {
   get: () => cy.get(standaloneSearchBoxComponent),
+  quanticSearchBoxInput: () =>
+    StandaloneSearchBoxSelectors.get().find('c-quantic-search-box-input'),
   input: (textarea = false) =>
     StandaloneSearchBoxSelectors.get().find(
-      textarea ? 'textarea[type="search"]' : 'input[type="search"]'
+      textarea
+        ? 'c-quantic-search-box-input textarea[type="search"]'
+        : 'c-quantic-search-box-input input[type="search"]'
     ),
   suggestionList: () =>
     StandaloneSearchBoxSelectors.get().find(
-      'c-quantic-search-box-suggestions-list li'
+      'c-quantic-search-box-input c-quantic-search-box-suggestions-list li'
     ),
   clearButton: () =>
-    StandaloneSearchBoxSelectors.get().find('.searchbox__clear-button'),
+    StandaloneSearchBoxSelectors.get().find(
+      'c-quantic-search-box-input button[name="Clear"]'
```

Comment:
it's a better practice to use a special attribute to select elements for cypress, we usually use the attribute `data-cy` this attribute only serves for cypress selectors, thus it reduces the chances of accidentally updating the `name` attribute in the HTML of the component breaking by this the current tests.